collect_app/src/androidTest/java/org/odk/collect/android/formentry/audit/TrackChangesReasonTest.java

```diff
@@ -123,7 +123,9 @@ public void openingAFormToEdit_andChangingAValue_andClickingSaveAndExit_andRotat
                 .clickSaveAndExitWithChangesReasonPrompt()
                 .enterReason("Something")
                 .rotateToLandscape(new ChangesReasonPromptPage("Track Changes Reason", rule))
-                .assertText("Something");
+                .assertText("Something")
+                .rotateToPortrait(new ChangesReasonPromptPage("Track Changes Reason", rule))
```

Comment:
I think this should be `closeSoftkeyboard()` instead just so we are checking the full "rotated" flow.

Reply:
Done.